sql/moz-fx-data-shared-prod/firefox_ios_derived/firefox_ios_first_28_day_retention_v1/query.sql

```diff
@@ -0,0 +1,36 @@
+WITH first_28_day_activity AS (
+  SELECT
+    client_id,
+    sample_id,
+    mozfun.bits28.retention(
```

Comment:
@soGaussian are you aware if this is the correct usage of this UDF? I recall you mentioning there are some udf's we could use for this.

Reply:
@fbertsch do you know if this udf is reliable and should be used for calculating retention information? Does it looks like I'm using it correctly here?